mantidproject/mantid
# Fix shortcuts in the Reflectometry GUI

Shortcuts to copy/cut/paste rows were not working in the Reflectometry GUI.

**To test:**

- Open `Interfaces` -> `Reflectometry` -> `ISIS Reflectometry`.
- Populate the table with some values, for example:

![reflectometry-table](https://cloud.githubusercontent.com/assets/8956985/23699546/73cd12ba-03e7-11e7-9d41-d9b3cc62a078.png)

- Check that you can copy/cut selected row(s) with Ctrl+c/Ctrl+x and paste them with Ctrl+v.

Fixes #19089.

Release notes: https://github.com/mantidproject/mantid/pull/19091/commits/b9ba05d55f73e81f3f64aa6d63de805caf4bb48d

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
